test/integration/controller/core/localqueue_controller_test.go

```diff
@@ -91,32 +91,60 @@ var _ = ginkgo.Describe("Queue controller", ginkgo.Ordered, ginkgo.ContinueOnFai
 	})
 
 	ginkgo.It("Should update conditions when clusterQueues that its localQueue references are updated", func() {
-		gomega.Eventually(func() []metav1.Condition {
+		gomega.Eventually(func() kueue.LocalQueueStatus {
 			var updatedQueue kueue.LocalQueue
 			gomega.Expect(k8sClient.Get(ctx, client.ObjectKeyFromObject(queue), &updatedQueue)).To(gomega.Succeed())
-			return updatedQueue.Status.Conditions
-		}, util.Timeout, util.Interval).Should(gomega.BeComparableTo([]metav1.Condition{
-			{
-				Type:    kueue.LocalQueueActive,
-				Status:  metav1.ConditionFalse,
-				Reason:  "ClusterQueueDoesNotExist",
-				Message: "Can't submit new workloads to clusterQueue",
+			return updatedQueue.Status
+		}, util.Timeout, util.Interval).Should(gomega.BeComparableTo(kueue.LocalQueueStatus{
+			Conditions: []metav1.Condition{
+				{
+					Type:    kueue.LocalQueueActive,
+					Status:  metav1.ConditionFalse,
+					Reason:  "ClusterQueueDoesNotExist",
+					Message: "Can't submit new workloads to clusterQueue",
+				},
 			},
 		}, util.IgnoreConditionTimestampsAndObservedGeneration))
 
+		emptyUsage := []kueue.LocalQueueFlavorUsage{
+			{
+				Name: flavorModelC,
+				Resources: []kueue.LocalQueueResourceUsage{
+					{
+						Name:  resourceGPU,
+						Total: resource.MustParse("0"),
+					},
+				},
+			},
+			{
+				Name: flavorModelD,
+				Resources: []kueue.LocalQueueResourceUsage{
+					{
+						Name:  resourceGPU,
+						Total: resource.MustParse("0"),
+					},
+				},
+			},
+		}
+
 		ginkgo.By("Creating a clusterQueue")
 		gomega.Expect(k8sClient.Create(ctx, clusterQueue)).To(gomega.Succeed())
-		gomega.Eventually(func() []metav1.Condition {
+		gomega.Eventually(func() kueue.LocalQueueStatus {
 			var updatedQueue kueue.LocalQueue
 			gomega.Expect(k8sClient.Get(ctx, client.ObjectKeyFromObject(queue), &updatedQueue)).To(gomega.Succeed())
-			return updatedQueue.Status.Conditions
-		}, util.Timeout, util.Interval).Should(gomega.BeComparableTo([]metav1.Condition{
-			{
-				Type:    kueue.LocalQueueActive,
-				Status:  metav1.ConditionFalse,
-				Reason:  "ClusterQueueIsInactive",
-				Message: "Can't submit new workloads to clusterQueue",
+			return updatedQueue.Status
+		}, util.Timeout, util.Interval).Should(gomega.BeComparableTo(kueue.LocalQueueStatus{
+			Conditions: []metav1.Condition{
+				{
+					Type:    kueue.LocalQueueActive,
+					Status:  metav1.ConditionFalse,
+					Reason:  "ClusterQueueIsInactive",
+					Message: "Can't submit new workloads to clusterQueue",
+				},
 			},
+			FlavorsReservation: emptyUsage,
+			FlavorUsage:        emptyUsage,
+			AvailableFlavors:   []kueue.ResourceFlavorReference{"model-c", "model-d"},
```

Comment:
Any reason not to do:
```suggestion
			AvailableFlavors:   []kueue.ResourceFlavorReference{flavorModelC, flavorModelD},
```
here and below?

Reply:
Done

Reply:
Yeah, that's a good point. Thanks for the review!